### PR TITLE
doc: add user guide and tutorial doc of mcs

### PR DESCRIPTION
**Pull request by jwcesign**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
doc: add user guide and tutorial doc of mcs

**Which issue(s) this PR fixes**:
Fixes #none

**Special notes for your reviewer**:
none

### CODE REVIEW

File: docs/tutorials/access-service-across-clusters.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+title: Access service acrosss clusters within native service name
```
Comment (XiShanYongYe-Chang) [inline docs/tutorials/access-service-across-clusters.md:2]: Do I feel that native service is more appropriate?

File: docs/tutorials/access-service-across-clusters.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+---
+
+In Karmada, the MultiClusterService can enable users to access services across clusters with the native service name, like `foo.svc`, with the aim of providing a seamless experience for users to access services across clusters.
```
Comment (XiShanYongYe-Chang) [inline docs/tutorials/access-service-across-clusters.md:5]: Is foo.svc the name of the service or the domain to access?

File: docs/tutorials/access-service-across-clusters.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+$ karmadactl get po -C member2
+NAME                    CLUSTER   READY   STATUS    RESTARTS   AGE
+curl-6894f46595-c75rc   member2   1/1     Running   0          15s
+```
+
+Later, we will run the curl command in this pod.
+
+## Deploy MultiClusterService and Service in Karmada
+
+Now, let's deploy Service and MultiClusterService in Karmada:
```
Comment (XiShanYongYe-Chang) [inline docs/tutorials/access-service-across-clusters.md:128]: Can here briefly describe this place in words? Now we are no need to propagate the service to the member cluster through PropagationPolicy.

File: docs/tutorials/access-service-across-clusters.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
+---
+title: Access service acrosss clusters within native service name
+---
+
+In Karmada, the MultiClusterService can enable users to access services across clusters with the native service name, like `foo.svc`, with the aim of providing a seamless experience for users to access services across clusters.
+
+This document provides an example of how to enable MultiClusterService for accessing service across clusters with native service name.
+
+## Prerequisites
```
Comment (XiShanYongYe-Chang) [inline docs/tutorials/access-service-across-clusters.md:9]: Should we explain the limitations of using this feature in this chapter, such as the requirement for interconnectivity of container networks between member clusters?

File: docs/userguide/service/multi-cluster-service-with-native-svc-access.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## API Object
+
+The MultiClusterService is an API in the Karmada networking API group. The current version is v1alpha1.
+
+You can check the FederatedHPA API specification [here](https://github.com/karmada-io/karmada/blob/65376b28d5037c27ff7ec0e56542c2a345d1a120/pkg/apis/networking/v1alpha1/service_types.go#L50).
+
+## What's next
+
+If you configure MultiClusterService, you may also want to consider how to connectt the network between clusters, such as [Submariner](../network/working-with-submariner).
```
Comment (XiShanYongYe-Chang) [inline docs/userguide/service/multi-cluster-service-with-native-svc-access.mdx:38]: typo: connectt

File: docs/tutorials/access-service-across-clusters.md (hunk 2 above)
Comment (jwcesign) [reply docs/tutorials/access-service-across-clusters.md:5]: should be domain name

**Comment by jwcesign** [comment]
/cc @XiShanYongYe-Chang 
Please take another look

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Comment by jwcesign** [comment]
/cc @RainbowMango 
for checking

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/service/multi-cluster-service-with-native-svc-access.mdx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Karmada will collect EndpointSlice resources from all target clusters, and sync them to the Karmada control plane.
+2. Karmada will sync the collected EndpointSlice resources to all target clusters, with attaching the EndpointSlice to the service.
+3. When users access through `foo.svc`, the underlying network will route the request to the backend pods in the multiple clusters.
+
+## API Object
+
+The MultiClusterService is an API in the Karmada networking API group. The current version is v1alpha1.
+
+You can check the FederatedHPA API specification [here](https://github.com/karmada-io/karmada/blob/65376b28d5037c27ff7ec0e56542c2a345d1a120/pkg/apis/networking/v1alpha1/service_types.go#L50).
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/service/multi-cluster-service-with-native-svc-access.mdx:34]:
```suggestion
You can check the MultiClusterService API specification [here](https://github.com/karmada-io/karmada/blob/65376b28d5037c27ff7ec0e56542c2a345d1a120/pkg/apis/networking/v1alpha1/service_types.go#L50).
```

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/service/multi-cluster-service-with-native-svc-access.mdx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+## What's next
+
+If you configure MultiClusterService, you may also want to consider how to connect the network between clusters, such as [Submariner](../network/working-with-submariner).
+
+For more information on MultiClusterService:
+* Read [Access serice across clusters within native service name](../../tutorials/access-service-across-clusters) to know how to use MultiClusterService.
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/service/multi-cluster-service-with-native-svc-access.mdx:41]:
```suggestion
* Read [Access service across clusters within native service name](../../tutorials/access-service-across-clusters) to know how to use MultiClusterService.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/access-service-across-clusters.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+      clusterNames:
+        - member1
+```
+
+After deploying, you can check the created pods:
+```sh
+$ karmadactl get po
+NAME                     CLUSTER   READY   STATUS              RESTARTS   AGE
+nginx-5c54b4855f-6sq9s   member1   0/1     Running             0          28s
+nginx-5c54b4855f-vp948   member1   0/1     Running             0          28s
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/access-service-across-clusters.md:75]:
```suggestion
nginx-5c54b4855f-6sq9s   member1   1/1     Running             0          28s
nginx-5c54b4855f-vp948   member1   1/1     Running             0          28s
```
:)

File: docs/resources/userguide/service/multiclusterservice/mcs-overview.png
Comment (RainbowMango) [inline docs/resources/userguide/service/multiclusterservice/mcs-overview.png:1]: Do you think the image source should be uploaded?
Same as below.

**Review by RainbowMango — COMMENTED** [review]
Nice work! Just some typos found.

File: docs/resources/userguide/service/multiclusterservice/mcs-overview.png
Comment (jwcesign) [reply docs/resources/userguide/service/multiclusterservice/mcs-overview.png:1]: Sure

File: docs/tutorials/access-service-across-clusters.md [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+
+In Karmada, the MultiClusterService can enable users to access services across clusters with the native service domain name, like `foo.svc`, with the aim of providing users with a seamless experience when accessing services across multiple clusters, as if they were operating within a single cluster.
+
+This document provides an example of how to enable MultiClusterService for accessing service across clusters with native service.
+
+## Prerequisites
+
+### Karmada has been installed
+
+We can install Karmada by referring to [Quick Start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/local-up-karmada.sh` script which is also used to run our E2E cases.
```
Comment (zhzhuang-zju) [inline docs/tutorials/access-service-across-clusters.md:13]: Is it necessary to inform users about hack/local-up-karmada.sh is used for e2e cases?

File: docs/resources/userguide/service/multiclusterservice/mcs-overview.drawio [hunk 10]
```diff
@@ -0,0 +1,108 @@
+<mxfile host="Electron" modified="2024-01-09T11:15:29.317Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/21.6.8 Chrome/114.0.5735.289 Electron/25.5.0 Safari/537.36" etag="ZryrKlNv4eWpEWFRcYkw" version="21.6.8" type="device">
```
Comment (zhzhuang-zju) [inline docs/resources/userguide/service/multiclusterservice/mcs-overview.drawio:1]: Are these codes the source code of your images? looks good~

Comment (jwcesign) [reply docs/resources/userguide/service/multiclusterservice/mcs-overview.drawio:1]: Yes, it's the source file of draw.io

File: docs/tutorials/access-service-across-clusters.md (hunk 9 above)
Comment (jwcesign) [reply docs/tutorials/access-service-across-clusters.md:13]: I copied from https://karmada.io/docs/next/userguide/scheduling/scheduler-estimator/#karmada-has-been-installed , I think it's ok

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**